frontend/src/components/employees/index.tsx

```diff
@@ -139,17 +137,41 @@ const addButton = (
 const Employees = () => {
   const staffList = useSelector(selectStaff);
   const dispatch = useDispatch();
+  const [isDeleteModalOpen, setDeleteModalState] = useState(false);
+  const [selectedStaff, setSelectedStaff] = useState({
+    id: 0,
+    firstName: '',
+    surname: '',
+    jobTitle: '',
+    contractedHours: 0,
+  });
+
+  const closeModal = () => {
+    setDeleteModalState(false);
+  };
+
+  const openDeleteModal = (staff: StaffData) => () => {
+    setDeleteModalState(true);
+    setSelectedStaff(staff);
+  };
+
+  const tableColumns = makeTableColumns(editUser, openDeleteModal);
 
   // Fetch data when component loads
   useEffect(() => {
-    dispatch(fetchStaff(getAuthenticationToken(), STAFF_FETCH_URL));
+    dispatch(fetchStaff(getAuthenticationToken()));
```

Comment:
Okay so this might not be a thing we have to fix in this PR but I also think that the frontend components shouldn't care about authentication if that makes sense? given many api calls will be using auth. The auth logic should be handled in one place where we define our api calls? Don't feel the need to change it here but something to consider

Reply:
That is something I think we can address as part of #67 so I think you can leave this just now

Reply:
eh, it's a quick fix so I just did it